Source/Cells/Text/FORMTextFieldCell.m

```diff
@@ -187,6 +187,7 @@ - (void)updateWithField:(FORMField *)field {
     self.textField.rawText         = [self rawTextForField:field];
     self.textField.info            = field.info;
     self.textField.styles          = field.styles;
+    self.textField.placeholder     = (!field.disabled) ? field.placeholder : nil;
```

Comment:
@jgorset you too 😊 
![giphy](https://cloud.githubusercontent.com/assets/1088217/14752536/14ba95b2-08d0-11e6-82a4-5d2aebc59aea.gif)